### ISSUE TITLE
Move away from github

### ISSUE THREAD

**Issue by francisco-core** [PrivacyLx/privacylx-issue-tracker#33]
We should move away from github into gitlab or [0xACAB's gitlab](https://0xacab.org/explore) or our self-hosted git.

Issues:
  - we can only have 3 contributors to the organization
  - micro$oft owns github
  - contributes to centralization

**Comment by anadahz** [comment]
[Gitea](https://gitea.io/en-us/) is also a good and lightweight alternative to Gitlab.

Have a look also to [Gitea compared to other Git hosting options](https://docs.gitea.io/en-us/comparison/).

**Comment by francisco-core** [comment]
Yup. I'm aware of gitea, but we have to consider if we are able to sustain more infrastructure or not.
I do have a gitea ansible role that we can use
